frontend/src/pages/ActivityReport/Pages/components/NextStepsRepeater.js

```diff
@@ -76,68 +97,112 @@ export default function NextStepsRepeater({
-          </FormGroup>
+              {blurStepValidations[index] || (errors[name]
+                && errors[name][index] && errors[name][index].note)
+                ? <ErrorMessage>Enter a next step</ErrorMessage>
+                : null}
+              <div
+                key={`${stepType}-next-step-flex-step-${index + 1}`}
+                className={`display-flex ${blurStepValidations[index]
+                  || (errors[name] && errors[name][index]
+                    && errors[name][index].note) ? 'blank-next-step' : ''}`}
+              >
+                <Textarea
+                  key={item.key}
+                  id={`${stepType}-next-step-${index + 1}`}
+                  className="height-10 minh-5 smart-hub--text-area__resize-vertical"
+                  name={`${name}[${index}].note`}
+                  type="text"
+                  defaultValue={item.note}
+                  inputRef={register({ required: 'Enter a next step' })}
+                  onBlur={({ target: { value } }) => validateStepOnBlur(value, index)}
+                  data-testid={`${name === 'specialistNextSteps' ? 'specialist' : 'recipient'}NextSteps-input`}
+                  style={{ height: !heights[index] ? `${DEFAULT_STEP_HEIGHT}px` : heights[index] }}
+                  onChange={(e) => onStepTextChanged(e, index)}
+                />
+                {canDelete ? (
+                  <Button
+                    className="margin-top-0"
+                    unstyled
+                    type="button"
+                    aria-label={`remove ${ariaName} ${index + 1}`}
+                    onClick={() => onRemoveStep(index)}
+                  >
+                    <FontAwesomeIcon className="margin-x-1" color="#000" icon={faTrash} />
+                    <span className="sr-only">
+                      remove step
+                      {' '}
+                      {index + 1}
+                    </span>
+                  </Button>
+                ) : null}
+              </div>
+            </FormGroup>
+            <FormGroup
+              key={`${stepType}-next-step-form-group-date-${index + 1}`}
+              className="margin-top-1 margin-bottom-3"
+              error={blurDateValidations[index] || (errors[name] && errors[name][index]
+                && errors[name][index].completeDate)}
+            >
+              <Label
+                htmlFor={`${name}[${index}].completeDate`}
+              >
+                {`When do you anticipate completing step ${index + 1}?`}
+                <span className="smart-hub--form-required font-family-sans font-ui-xs text-secondary-dark">
+                  {' '}
+                  *
+                </span>
+              </Label>
+              {blurDateValidations[index]
+                || (errors[name] && errors[name][index]
+                  && errors[name][index].completeDate)
+                ? <ErrorMessage>Enter a complete date</ErrorMessage>
+                : null}
```

Comment:
I would double check on the message here. I would think we'd want it to say "Enter a valid date" like the other date pickers.

Reply:
Updated error message to read "Enter a valid  date".